Sources/Guaka/Flag/Flags.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Flag.swift
+//  CommandLine
+//
+//  Created by Omar Abdelhafith on 30/10/2016.
+//
+//
+
+public struct Flags {
```

Comment:
This type provides easier ways to access the flags finally passed to a command,
It basically encapsulate access to a `[String: Flag]`